modules/cloudfront.md

```diff
@@ -0,0 +1,209 @@
+---
+layout: module
+title: CloudFront
+image: assets/img/cloudfront.png
+description: Create and manage CloudFront web distributions and run invalidations
+---
+Overview
+--------
+Cumulus makes using CloudFront easier and cleaner! Easily configure web distributions and update cache behaviors! Run commonly used invalidations! Read the following sections to learn about configuring your CloudFront environment with Cumulus. Example configuration can be found in the [Cumulus repo](https://github.com/lucidsoftware/cumulus).
+
+Distribution Definition
+-----------------------
+
+Cumulus currently only supports managing CloudFront [Web Distributions](http://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/distribution-web.html) because these are the only distributions we use at this time at Lucid. If you have additional needs, submit a pull request!
+
+Each distribution is defined in its own file, and the folder distributions are located in is [configurable](#configuration). In other Cumulus modules, you've probably seen that managed assets get their names from the names of the files in which they are defined. However, because CloudFront distributions are only identifiable by their ID in AWS, file names for distributions are arbitrary. The only thing to note is that the file name is the name that Cumulus will use to refer to your hosted zone in its output, and as input into its command line interface (minus the `.json`, of course). Distributions are JSON objects with the following attributes:
+
+* `id` - the id of your distribution. If omitted, a new distribution will be created with the configuration
```

Comment:
If omitted, a new distributions will be created next time the configuration is synced